### PR TITLE
Ava UI: Add Open Mod Dir Buttons

### PR DESCRIPTION
**Pull request by IsaacMarovitz**
Adds back the old buttons... even though they really shouldn't be added back.

### CODE REVIEW

File: src/Ryujinx.Ava/UI/Windows/CheatWindow.axaml.cs [hunk 1]
```diff
@@ -119,5 +124,32 @@ public void Save()
...
+                catch (Exception ex)
+                {
+                    Dispatcher.UIThread.InvokeAsync(async () =>
+                    {
+                        await ContentDialogHelper.CreateErrorDialog(LocaleManager.Instance.UpdateAndGetDynamicValue(
+                            LocaleKeys.DialogLoadFileErrorMessage,
+                            ex,
+                            cheatsPath));
+                    });
+                    return;
```
Comment (AcK77) [inline src/Ryujinx.Ava/UI/Windows/CheatWindow.axaml.cs:148]:
```suggestion

                    return;
```

**Review by AcK77 — APPROVED** [review]

File: src/Ryujinx.Ava/UI/Windows/CheatWindow.axaml.cs [hunk 2]
```diff
@@ -119,5 +124,33 @@ public void Save()
 
             Close();
         }
+
+        public void OpenCheatFolder()
+        {
+            string modsBasePath = ModLoader.GetModsBasePath();
+            string cheatsPath = Path.Combine(modsBasePath, "contents", TitleId, "Cheats");
```
Comment (TSRBerry) [inline src/Ryujinx.Ava/UI/Windows/CheatWindow.axaml.cs:131]:
```suggestion
            string cheatsPath = Path.Combine(modsBasePath, "contents", TitleId, "cheats");
```

Why not restore the previous buttons? They were better imo.

**Comment by TSRBerry** [comment]
I'd still prefer to have the old buttons back instead of this.

**Review by gdkchan — APPROVED** [review]
Nice.

**Pull request merged**